build-output-api/isr-api-routes/.vercel/output/functions/revalidate.func/index.js

```diff
@@ -0,0 +1,71 @@
+const https = require('https')
+
+// The bypass token should be a build-time randomly generated string of at least 32 characters.
+// This value should only be exposed in the function config and inside of the function itself.
+// DO NOT expose this value on the client-side: a malicious user could trigger boundless revalidations.
+const bypassToken = '87734ad8259d67c3c11747d3e4e112d0'
+
+// This authToken is a trivial example of authentication, but any strong authentication method will work.
+// DO NOT allow unauthenticated users to trigger revalidation: a malicious user could trigger boundless revalidations.
+const authToken = 'a13f94f6-a441-47ca-95fc-9f44f3450295';
+
+function revalidate(host, path) {
```

Comment:
I assume this can be done in a nextjs API route as well right? (as long as the token and validation mechanism are properly secured). So basically I can force revalidate the cache with the `x-prerender-revalidate`, and that this should only be done by authorized entity

Reply:
API routes in Next.js do not currently leverage our `Prerender` output type, to leverage `stale-while-revalidate` behavior the `cache-control` header must be set manually so do not currently support On-Demand Revalidate

Reply:
Would it work if the API route set the right cache header?